packages/trader/src/sass/app/_common/drawer/contract-drawer.scss

```diff
@@ -73,6 +73,32 @@ $header-height: 4em;
 
 /** @define .contract-card; weak */
 .contract-card {
+    &__market-closed {
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        flex-direction: column;
+        padding: 0;
+        background-image: var(--contract-gradient-danger);
+        z-index: 1000;
+    }
+    &__footer {
+        margin-bottom: 0.5em;
```

Comment:
```suggestion
        margin-bottom: 0.5rem;
```

Reply:
I didn't touch this section (same for the other two suggestions) but will apply them.